pkg/kubectl/cmd/drain.go

```diff
@@ -266,40 +267,36 @@ func (o *DrainOptions) deleteOrEvictPodsSimple() error {
 	return err
 }
 
-func (o *DrainOptions) getController(sr *api.SerializedReference) (interface{}, error) {
-	switch sr.Reference.Kind {
+func (o *DrainOptions) getController(pod *api.Pod, controllerRef *metav1.OwnerReference) (interface{}, error) {
```

Comment:
Do a sanity check to make sure controllerRef isn't `nil`, and pass namespace instead of `pod` since we don't need the whole `pod`.